tests/functional/workloads/cnv/test_vm_cloning_ops.py

```diff
@@ -62,3 +62,60 @@ def test_vm_clone(self, cnv_workload, clone_vm_workload, setup_cnv):
             ), f"Failed: MD5 comparison between source {vm_obj.name} and cloned {clone_obj.name} VMs"
             run_dd_io(vm_obj=clone_obj, file_path=file_paths[1])
             clone_obj.stop()
+
+    @workloads
+    @pytest.mark.polarion_id("OCS-6299")
+    def test_vm_snapshot_ops(
```

Comment:
Need to change test script accordingly as it has snapshot tc as well

Reply:
ack, renamed it